lib/structures/Guild.js

```diff
@@ -130,7 +133,18 @@ class Guild extends Base {
     }
 
     get iconURL() {
-        return this.icon ? `${CDN_URL}/icons/${this.id}/${this.icon}.jpg` : null;
+        return this.icon ? `${CDN_URL}/icons/${this.id}/${this.icon}.${this._client.options.defaultImageFormat}${this._client.options.defaultImageSize == 128 ? "" : `?size=${this._client.options.defaultImageSize}`}` : null;
+    }
+
+    /**
+    * Get the guild's icon with the given format and size
+    * @arg {String} [format=default] The filetype of the avatar (jpg, png, gif, or webp)
+    * @arg {Number} [size=default] The size of the avatar (128, 256, 512, 1024, 2048)
+    */
+    dynamicIconURL(format, size) {
+        if (format === undefined || !ImageFormats.includes(format.toLowerCase())) format = this._client.options.defaultImageFormat;
+        if (size === undefined || !ImageSizes.includes(size)) size = this._client.options.defaultImageSize;
```

Comment:
Use `if(!~array.indexOf(item))` instead of `if(!array.includes(item))`

If statements should be formatted like the rest of the lib code
```
if(bool) {
    // thing
}
```